### PR TITLE
Putting it together

### PR DESCRIPTION
**Pull request by Sarahtonein**

### CODE REVIEW

**Comment by Sarahtonein** [comment]
> Can you give a quick overview of what you are doing and trying to achieve and your thoughts

At the moment I am making some changes to how we tag / alert people to events happening in the next 24 hours and I need to include the draft proposal stuff you mentioned the other day.

I am working on setting up discord => snapshot as well. Looking into this further today.

Outside of that, the gov-bot / emotes / update branches have been merged into here and completed / ready to review pending the other changes mentioned above. I can close this PR for now until the other changes have been made.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '1'
```
Comment (Balugrizzly) [inline docker-compose.yml:1]: Why are you using version 1 ?

File: docker-compose.yml
Comment (Balugrizzly) [inline docker-compose.yml:1]: Can you add a build step here

File: emojis/contributors.json
Comment (Balugrizzly) [inline emojis/contributors.json:1]: FMT would be nice.

Also combining this into the config might be a consideration. UP to you.

**Review by Balugrizzly — CHANGES_REQUESTED** [review]
This is a partial review.
I have stopped because i would highly recommend that you split this up into dedicated PR's with clear structure.

**Comment by Balugrizzly** [comment]
Btw Pipenv automatically loads env vars.

**Comment by Sarahtonein** [comment]
> This is a partial review. I have stopped because i would highly recommend that you split this up into dedicated PR's with clear structure.

No worries will do this :)

File: docker-compose.yml (hunk 1 above)
Comment (Sarahtonein) [reply docker-compose.yml:1]: > Why are you using version 1 ?

Was not sure what version # to use

File: docker-compose.yml
Comment (Sarahtonein) [reply docker-compose.yml:1]: > Can you add a build step here

Will do.

File: emojis/contributors.json
Comment (Sarahtonein) [reply emojis/contributors.json:1]: > FMT would be nice.
> 
> Also combining this into the config might be a consideration. UP to you.

I'll have a look at this :)

**Pull request closed without merging**